CHANGELOG

```diff
@@ -61,6 +61,11 @@ Qtile X.X.X, released XXXX-XX-XX:
         - new OpenWeather widget to replace YahooWeather
         - new format attribute for WindowName widget
         - new max_chars attribute for WindowName widget
+        - added `cmd_keep_above` and `cmd_keep_below` to Windows so they can be
+          kept above/below other windows permanently; calling the functions
+          twice removes the flag
+        - added `cmd_raise_client` and `cmd_lower_client` to Windows so they
```

Comment:
I chose `cmd_move_up` and `cmd_move_down` now. I assume `cmd_keep_above` etc.

Reply:
Maybe `cmd_lift` and `cmd_lower` might be even better?

Reply:
I like move_down and move_up

Reply:
Wait, no, this might conflict with the logic behind #1875 some day. What do you think of move_below and move_above? That would be consistent with the two other methods.

Reply:
Renamed.